examples/tutorials/next.md

```diff
@@ -269,20 +295,25 @@ return (
 );
 ```
 
+### Add some styles
+
+Let's add some basic styles to make the app look nicer. Update your
+`app/globals.css` file with the
+[styles from this file](https://raw.githubusercontent.com/denoland/tutorial-with-next/refs/heads/main/src/app/globals.css).
+
 ## Run the app
 
-Now you can run the app with `deno task dev` and visit `http://localhost:3000`
-in your browser to see the list of dinosaurs. Click on a dinosaur to see more
+Now you can run the app with `deno run dev` and visit `http://localhost:3000` in
+your browser to see the list of dinosaurs. Click on a dinosaur to see more
 details!
 
-![demo of the app](./images/how-to/next/dinoapp.gif)
+## Deploy the app
 
-## Deploy your app
+Now that you have your working Next.js app, you can deploy it to the web with
+Deno Deploy<sup>EA</sup>.
 
-Now that you have a Next.js app, you can deploy it to the web with Deno
-Deploy<sup>EA</sup>. Deno Deploy requires your code to be hosted on GitHub to
-access and deploy it, so first you'll need to create a GitHub repository and
-push your app there.
+Deno Deploy requires your code to be hosted on GitHub to access and deploy it,
```

Comment:
This isn't entirely correct since can just push content directly to an app via the CLI or deployctl. Perhaps something like:

```suggestion
For the best experience and automated workflow, Deno Deploy connects to your code on GitHub to access and deploy it,
```